subcommittees/mentoring/proposal-final-exercise-of-instructor-training.md

```diff
@@ -0,0 +1,41 @@
+# Checkout Session for Instructor Training
+
+**This is a rough draft.**
+
+## Terminology
+
+-   Instructor Trainer: someone appointed by the Software Carpentry
+    Steering Committee to lead a Instructor Training event.
+-   Trainee: anyone that attended a Software Carpentry Instructor Training event.
+-   [Pre-Checkout discussion session](#pre-checkout-discussion-session):
+    meeting between a Instructor Trainer and some trainees
+    to discuss one of Software or Data Carpentry lessons.
+-   [Checkout session](#checkout-session): meeting between a Instructor Trainer
+    and a trainee to conclude Instructor Training.
+
+## Pre-Checkout Discussion Session
+
```

Comment:
You sketch it out in the Terminology section, but I'd just reiterate here what this discussion is about and [why we have it in person][1].

[1]: https://github.com/swcarpentry/board/pull/68#discussion_r47819007